README.md

```diff
@@ -24,15 +25,15 @@ void setup() {
 }
 
 void loop() {
-  boltiot.CheckPoll(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
-/*This function needs to be called reglarly. Calling the CheckPoll function once every seconds is required*/
+  boltiot.processPushDataCommand(digitalRead(2)); //Send data to the bolt cloud, when the bolt polls the arduino,for data.
+/*This function needs to be called reglarly. Calling the processPushDataCommand function once every seconds is required*/
```

Comment:
@Vinayakj009 Typo in comment. `once every second`

Reply:
Correcting it